package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@batou.dev/eslint-config-typescript",
-  "version": "0.6.1",
+  "version": "0.6.2",
```

Comment:
Could you revert the version to `0.6.1` please? I'll handle a proper release afterwards.